clicommand/agent_start.go

```diff
@@ -1566,3 +1588,11 @@ func envHasKey(key string) bool {
 	return ok
 
 }
+
+func cancelGracePeriodDuration(cancelGracePeriod int) time.Duration {
+	if cancelGracePeriod < 10 {
+		return 10 // minimum 10 seconds for forceful shutdown
```

Comment:
The function's return type is `time.Duration`, but `time.Duration(10)` means 10 nanoseconds. If we keep this function it ought to return a `time.Duration` that represents the intended duration (as in, multiply by `time.Second` within).